src/Slices/EditInstance/Data/CommandManager.test.ts

```diff
@@ -42,38 +42,36 @@ test("GIVEN getBodyV1 WHEN changing optional attributes THEN generates correct b
   expect(body.attributes.attr4).toEqual(42);
 });
 
-// TODO make PATCH V2 ready
-// test("GIVEN getBodyV2 WHEN changing optional attributes THEN generates correct body", () => {
-//   const service_entity = "test-entity";
-//   const service_version = 4;
+//TODO make PATCH V2 ready
+test("GIVEN getBodyV2 WHEN changing optional attributes THEN generates correct body", () => {
+  const service_entity = "test-entity";
+  const service_version = 4;
 
-//   const fields = [
-//     { ...Field.text, name: "attr1", isOptional: false, type: "string" },
-//     { ...Field.text, name: "attr2", isOptional: true, type: "string?" },
-//     { ...Field.bool, name: "attr3", isOptional: true, type: "bool?" },
-//     { ...Field.number, name: "attr4", isOptional: true, type: "int?" },
-//   ];
-//   const attributes = {
-//     attr2: null,
-//     attr3: true,
-//     attr4: "42",
-//   };
+  const fields = [
+    { ...Field.text, name: "attr1", isOptional: false, type: "string" },
+    { ...Field.text, name: "attr2", isOptional: true, type: "string?" },
+    { ...Field.bool, name: "attr3", isOptional: true, type: "bool?" },
+    { ...Field.number, name: "attr4", isOptional: true, type: "int?" },
+  ];
 
-//   const body = getBodyV2(
-//     fields,
-//     currentAttributes,
-//     attributes,
-//     service_entity,
-//     service_version
-//   );
-//   expect(body.edit[0]).toEqual({
-//     edit_id: "test-entity_version=4",
-//     operation: "replace",
-//     target: ".",
-//     value: {
-//       attr2: "",
-//       attr3: null,
-//       attr4: 42,
-//     },
-//   });
-// });
+  //to simulate actual behavior after refactor attr1 was added as from now on all of the fields are passed to getBodyV2 function in the flow.
```

Comment:
I think, if I remember well, the reason for one attribute missing, is that the property itself can actually be removed. And that's different from setting it to undefined/null.

Reply:
okay, make sense, I'll adjust

Reply:
I had to leave attr1 as field for that attr is required, but I deleted other, attr2.